blob/drivertest/drivertest.go

```diff
@@ -2087,24 +2094,30 @@ func testSignedURL(t *testing.T, newHarness HarnessMaker) {
 	// PUT the blob. Try with all URLs, only putURL should work.
 	for _, test := range []struct {
 		urlMethod   string
+		contentType string
 		url         string
 		wantSuccess bool
 	}{
-		{http.MethodGet, getURL, false},
-		{http.MethodDelete, deleteURL, false},
-		{http.MethodPut, putURL, true},
+		{http.MethodGet, "", getURL, false},
+		{http.MethodDelete, "", deleteURL, false},
+		{http.MethodPut, goodContentType, putURL, true},
+		{http.MethodPut, badContentType, putURL, false},
+		{http.MethodPut, "", putURL, false},
```

Comment:
This only tests using a content type in the URL. Do you think it's worth adding a second putURL with ContentType="" and adding testcases with that?